README.md

```diff
@@ -1,3 +1,125 @@
 # IONOS Event Notification Handler
 
 Event Notification Handler
+
+For more information, please visit []().
+
+## Installation & Usage
+
+### Requirements
+
+PHP 7.4 and later.
+Should also work with PHP 8.0.
+
+### Composer
+
+To install the bindings via [Composer](https://getcomposer.org/), add the following to `composer.json`:
+
+```json
+{
+  "repositories": [
+    {
+      "type": "vcs",
+      "url": "https://github.com/ionos-productivity/ionos-mail-notification-api-client.git"
+    }
+  ],
+  "require": {
+    "ionos-productivity/ionos-mail-notification-api-client": "*@dev"
+  }
+}
+```
+
+Then run `composer install`
+
+### Manual Installation
+
+Download the files and include `autoload.php`:
+
+```php
+<?php
+require_once('/path/to/OpenAPIClient-php/vendor/autoload.php');
+```
+
+## Getting Started
+
+Please follow the [installation procedure](#installation--usage) and then run the following:
+
+```php
+<?php
+require_once(__DIR__ . '/vendor/autoload.php');
+
+
+
+
+$apiInstance = new IONOS\MailNotificationAPI\Client\Api\EventAPIApi(
+    // If you want use custom http client, pass your client which implements `GuzzleHttp\ClientInterface`.
+    // This is optional, `GuzzleHttp\Client` will be used as default.
+    new GuzzleHttp\Client()
+);
+$brand = 'brand_example'; // string
+$wipeMessage = new \IONOS\MailNotificationAPI\Client\Model\WipeMessage(); // \IONOS\MailNotificationAPI\Client\Model\WipeMessage
+
+try {
+    $apiInstance->processRemoteWipeFinishedEvent($brand, $wipeMessage);
+} catch (Exception $e) {
+    echo 'Exception when calling EventAPIApi->processRemoteWipeFinishedEvent: ', $e->getMessage(), PHP_EOL;
+}
+
+```
+
+## API Endpoints
+
+All URIs are relative to *https://API_HOST/easynextcloud-mail-notification*
+
+Class | Method | HTTP request | Description
+------------ | ------------- | ------------- | -------------
+*EventAPIApi* | [**processRemoteWipeFinishedEvent**](docs/Api/EventAPIApi.md#processremotewipefinishedevent) | **POST** /event/{brand}/remote-wipe-finished | Nextlcoud user gets informed that a remote wipe for a specific device has finished
+*EventAPIApi* | [**processRemoteWipeStartedEvent**](docs/Api/EventAPIApi.md#processremotewipestartedevent) | **POST** /event/{brand}/remote-wipe-started | Nextlcoud user started a remote wipe for a specific device
```

Comment:
typo:
Mom: We have a Cloud Storage at home
Cloud Storage at home: `Nextlcoud`

Reply:
Autogenerated via openapi definition.. ..we can only communicate it to  api developers #sad